scripts/build-release-artifacts.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script automates the building of artifacts for a release.
+#
+# Reference:  docs/releasing.md
+
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+RELEASE_TAG="${1-}"
+if [[ ! "${RELEASE_TAG}" ]]; then
+  >&2 echo "usage: $0 <release tag of the form v[0-9]+.[0-9]+.[0-9]+>"
+  exit 1
+fi
+
+ROOT_DIR="$(cd "$(dirname "$0")/.." ; pwd)"
+RELEASE_VERSION="${RELEASE_TAG:1:${#RELEASE_TAG}-1}"
+
+pushd "${ROOT_DIR}"
+  # Build release artifacts for kubefedctl
+  make kubefedctl HOST_OS=linux
+  make kubefedctl HOST_OS=darwin
```

Comment:
HOST_OS is intended to specify the target of the symlink.
as you don't need the symlink here, it's more straightforward to do `make bin/kubefedctl-linux bin/kubefedctl-darwin`